components/31-molecules/partner-block/partner-block.config.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  status: 'alpha',
+  context: {
+    title: 'Partners',
+    description: 'Hier komt een begeleidende tekst bij het logo.'
```

Comment:
Add some information about the fact that the logo's in this blok are not fixed ?

Reply:
Done.